bot/converters.py

```diff
@@ -574,6 +574,55 @@ async def convert(self, ctx: Context, arg: str) -> t.Optional[dict]:
                 raise e
 
 
+class DayDuration(Converter):
+    """
+    Convert a string representing day time (hours and minutes) to a UTC datetime object.
+
+    The hours and minutes would be combined with UTC day if no 'am' or 'pm' is passed with
+    the string, then it is assumed that the time is in 24 hour format.
```

Comment:
This sentence doesn't make grammatical sense. I think you're missing a period (`.`) after the "`...with UTC day`", i.e. "`...with UTC day. If no 'am' or 'pm' is passed...`"